sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ConditionEvaluatesToConstant.cs

```diff
@@ -1236,41 +1236,111 @@ public static void M(MyStructWithOperator a)
             }
         }
 
-        public class NullableCases
+    public class NullableCases
+    {
+      void Case1()
```

Comment:
is there any change here in `Case1()`? if there's only whitespace changes, please revert